tools/git-ritual.sh

```
@@ -0,0 +1,156 @@
+echo ""
+
+# Commit type selection
+echo -e "${CYAN}Select commit type (Conventional Commits standard):${NC}"
+echo "  1) feat     — New feature or capability"
+echo "  2) fix      — Bug fix or correction"
+echo "  3) docs     — Documentation only"
+echo "  4) chore    — Maintenance, brain artifacts, EOD save"
+echo "  5) refactor — Code restructure without behaviour change"
+echo "  6) ci       — Continuous integration / deployment changes"
+echo ""
+read -p "Enter number [1-6]: " type_choice
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)